src/forms/FormFieldDropdownSingleSelection/FormFieldDropdownSingleSelectionTypes.tsx

```diff
@@ -1,5 +1,5 @@
-import { PopperProps } from "@material-ui/core/Popper";
 import { MosaicLabelValue } from "@root/types";
+import { PopperProps } from '@mui/material/Popper';

```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/simpleviewinc/sv-mosaic/pullRequest?prid=9520170)
```suggestion
import { PopperProps } from "@mui/material/Popper";
```